openstack/poller/values.yaml

```diff
@@ -31,7 +31,7 @@ poller:
   printMessage: false
   rebuildDSN: false
   rhea:
-  queueName: 
+  queueName:
```

Comment:
Sure this shouldn't be nested inside the `rhea:` key?

Reply:
see https://github.com/sapcc/helm-charts/blob/master/openstack/poller/templates/_config.yaml.tpl#L72

Reply:
You are right.... I did not actually change this value, and my IDE just removed an unneeded space at this line.
I will go ahead and nest and then merge.... but looking at this values file.... there are also other things wrong. Someone likely needs to go through this chart and improve